<application>
  <component name="AppStorage">
    <option name="lastReplacementTargetLanguage" value="CHINESE" />
    <histories>
      <item value="search Product By Page" />
      <item value="search Product Page" />
      <item value="Attribute" />
      <item value="Search Product" />
      <item value="search" />
      <item value="expiration" />
      <item value="expire End Time" />
      <item value="uploadvideo" />
      <item value="upload Video" />
      <item value="sign Direct" />
      <item value="convert To Store Id" />
      <item value="clerk Token Service" />
      <item value="after" />
      <item value="Conditionally executed code should be reachable" />
      <item value="Noncompliant Code Example" />
      <item value="Conditional expressions which are always true or false can lead to dead code. Such code is always buggy and should never be used in production." />
      <item value="get Buy Button Text" />
      <item value="buy Button Status" />
      <item value="未到销售日期" />
      <item value="SELLING" />
      <item value="OFF" />
      <item value="SHELF" />
      <item value="下架" />
      <item value="is Sell" />
      <item value="buy Button Text" />
      <item value="find Specify Date Price" />
      <item value="price" />
      <item value="today" />
      <item value="todayprice" />
      <item value="find Today Price" />
      <item value="find Current Price" />
      <item value="stock Num Status" />
      <item value="stock Num Switch" />
      <item value="default Stock Num" />
      <item value="use Default Inv Status" />
      <item value="Flag" />
      <item value="stock Num" />
      <item value="stock Num Flag" />
      <item value="bit" />
      <item value="Deduction" />
      <item value="bit Deduction" />
      <item value="slots" />
      <item value="10922" />
      <item value="Attempted" />
      <item value="Invocation of init method failed" />
      <item value="Invocation" />
      <item value="milliseconds" />
      <item value="Expected" />
      <item value="unexpectedly" />
      <item value="Can not read response from server" />
    </histories>
    <option name="languageScores">
      <map>
        <entry key="CHINESE" value="3485" />
        <entry key="ENGLISH" value="3250" />
        <entry key="CHINESE_TRADITIONAL" value="2" />
        <entry key="ICELANDIC" value="1" />
        <entry key="DANISH" value="3" />
        <entry key="GERMAN" value="8" />
        <entry key="FRENCH" value="74" />
        <entry key="FILIPINO" value="5" />
        <entry key="DUTCH" value="2" />
        <entry key="CZECH" value="2" />
        <entry key="ROMANIAN" value="1" />
        <entry key="MALAY" value="14" />
        <entry key="PORTUGUESE" value="19" />
        <entry key="TURKISH" value="1" />
        <entry key="SPANISH" value="8" />
        <entry key="ITALIAN" value="4" />
        <entry key="HINDI" value="5" />
        <entry key="INDONESIAN" value="1" />
      </map>
    </option>
  </component>
  <component name="Settings">
    <option name="ignoreRegExp" value="[-_]" />
    <option name="overrideFont" value="true" />
    <option name="phoneticFontFamily" value="Lucida Sans Unicode" />
    <option name="primaryFontFamily" value="Microsoft YaHei UI" />
    <option name="selectTargetLanguageBeforeReplacement" value="true" />
    <option name="showExplanation" value="true" />
    <option name="targetLanguageSelection" value="PRIMARY_LANGUAGE" />
    <option name="translator" value="ai.youdao" />
    <option name="youdaoTranslateSettings">
      <youdao-translate>
        <option name="appId" value="72daf72feb661e19" />
      </youdao-translate>
    </option>
  </component>
</application>